backend/Dockerfile

```diff
@@ -1,26 +1,29 @@
 FROM ubuntu:20.04 as build
 
-RUN apt-get update \
-    && apt-get install --assume-yes --no-install-recommends curl python3-pip \
+RUN apt-get update --quiet --quiet \
+    && apt-get install --quiet --quiet --assume-yes --no-install-recommends \
+    curl python3-pip < /dev/null > /dev/null \
```

Comment:
`< /dev/null` just closes standard input for the `apt-get` command. Does it actually make it less verbose? Also, with `--quiet --quiet` we probably don't want `> /dev/null` as well, that should be redundant.

Ditto for the other `apt-get`.

Reply:
Yes, it makes it less verbose then I was able to do with `--quiet`.

Reply:
`< /dev/null` removed but I left stdout redirection `> /dev/null`.